lib/commons/aria/get-role-type.js

```diff
@@ -1,21 +1,23 @@
 import standards from '../../standards';
+import AbstractVirtualNode from '../../core/base/virtual-node/abstract-virtual-node';
 
 /**
  * Get the "type" of role; either widget, composite, abstract, landmark or `null`
  * @method getRoleType
  * @memberof axe.commons.aria
  * @instance
- * @param {String} role The role to check
+ * @param {String|Node|Element} role The role to check, or element to check the role of
  * @return {Mixed} String if a matching role and its type are found, otherwise `null`
  */
 function getRoleType(role) {
-  const roleDef = standards.ariaRoles[role];
-
-  if (!roleDef) {
-    return null;
+  if (
+    role instanceof AbstractVirtualNode ||
+    (window?.Node && role instanceof window.Node)
+  ) {
+    role = axe.commons.aria.getRole(role);
```

Comment:
I didn't noticed that. Have I really? I used instanceof here because that's what we usually do when we want to check nodes. There are lots of `instanceof AbstractVirtualNode` calls.. and a couple `instanceof window.Node` calls too.

Looking at where/how this function has been used, it's going to get `null` passed into it a good bit too, since getRole() can return null. I'll update it to reflect that.